### DISCUSSION TITLE
Unable to set Journal Buffer Size through CRD

### DISCUSSION THREAD

**Discussion by jlplasce** [arkmq-org/arkmq-org-broker-operator#1035] [Q&A] [answered]
I am unable to find a valid location in the CRD to define the "\<journal-buffer-size\>" entry. 
I attempted:
```
brokerProperties:
  - journalBufferSize=xxxxxxx
```

But I do not see it appearing on the broker.xml or as an environment variables anywhere. Is this setting available elsewhere in the CRD or is this not the correct place/naming?

Operator version: 1.2.4
Artemis: 2.35.0

**Comment by brusdev** [comment]
@jlplasce the only way to check settings specified via broker properties is to query the management API, i.e. to query the JournalBufferSize from the broker container shell:
```
curl -u $AMQ_USER:$AMQ_PASSWORD http://$HOSTNAME:8161/console/jolokia/read/org.apache.activemq.artemis:broker=\"amq-broker\"/JournalBufferSize
```

**Reply by jlplasce** [reply]
When doing this, I dont see my updated value in place, I see the default "501760" still referenced. Is my location in the properties correct? does it need to be a capital "J" at the beginning of the property?

**Reply by brusdev** [reply, marked as answer]
There are 2 broker properties to set the journal buffer size:
- journalBufferSize_AIO for AIO journal
- journalBufferSize_NIO for NIO journal

The default journal type on Kubernetes is NIO

**Reply by jlplasce** [reply]
Thank you , that seems to have worked, I didnt realize there was a nuance to define "_AIO" or "_NIO" to get it to be registered
